coupon.go

```diff
@@ -31,12 +45,16 @@ type CouponParams struct {
 }
 
 type CouponInput struct {
-	Name               string           `json:"name,omitempty"`
-	Code               string           `json:"code,omitempty"`
-	AmountCents        int              `json:"amount_cents,omitempty"`
-	AmountCurrency     Currency         `json:"amount_currency,omitempty"`
-	Expiration         CouponExpiration `json:"expiration,omitempty"`
-	ExpirationDuration int              `json:"expiration_duration,omitempty"`
+	Name               string                   `json:"name,omitempty"`
+	Code               string                   `json:"code,omitempty"`
+	AmountCents        int                      `json:"amount_cents,omitempty"`
+	AmountCurrency     Currency                 `json:"amount_currency,omitempty"`
+	Expiration         CouponExpiration         `json:"expiration,omitempty"`
+    ExpirationDate     string                   `json:"expiration_date,omitempty"`
```

Comment:
I don't know why exactly, but it seems there is an indentation issue in this file

Reply:
Not sure :) , there is 1 tabulation before `Name`, `Code`... and 4 spaces before `ExpirationDate`, `PercentageRate` and so on

Reply:
You can see it in the raw view of the file: https://raw.githubusercontent.com/getlago/lago-go-client/7ae79a219eded28e763c79b90806c34b53d7ef59/coupon.go

Reply:
hmm, strange, in my IDE looks aligned :/ Will fix it:) Thanks for catching

Reply:
Just fixed it in text editor :D My IDE is broken, let's blame him:)